sdks/python/apache_beam/examples/snippets/snippets_test.py

```diff
@@ -470,6 +470,13 @@ def test_model_textio(self):
         ['aa', 'bb', 'bb', 'cc', 'cc', 'cc'],
         self.get_output(result_path, suffix='.csv'))
 
+  def test_model_datastoreio(self):
+    # We cannot test datastoreio functionality in unit tests therefore we limit
+    # ourselves to making sure the pipeline containing Datastore read and write
+    # transforms can be built.
+    # TODO(vikasrk): Expore using Datastore Emulator.
+    self.assertEqual(None, snippets.model_datastoreio())
```

Comment:
What are you checking here? Would not it raise an exception if it fails in some way unexpected?